book/01-guided-tour.html

```diff
@@ -177,22 +189,47 @@ <h1>Functions and Type Inference</h1>
       <code>int -&gt; int</code> is a function type, in this case
       indicating a function that takes an <code>int</code> and
       returns an <code>int</code>. We can also write functions that
-      take multiple arguments. (Note that the following example
-      will not work if you haven't opened
-      <code>Core_kernel</code> as was suggested
-      earlier.)<idx>multi-argument
-      functions</idx><idx>functions/with multiple
-      arguments</idx></p>
+      take multiple arguments. (Reminder: Don't forget to open <code>Base</code>,
```

Comment:
shouldn't the `open` also be within the <code> tag here?

Reply:
Sure. Done.